src/sqltokenizer.rs

```diff
@@ -286,6 +289,21 @@ impl<'a> Tokenizer<'a> {
                         }
                     }
                 }
+                x @ 'x' | x @ 'X' => {
```

Comment:
Which dialect supports the lower-case 'x'?

Reply:
PostgreSQL, at least. I've added a comment to this effect.